srcpkgs/diffuse/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'diffuse'
+pkgname=diffuse
+version=0.4.8
+revision=1
+archs=noarch
+build_style=fetch
```

Comment:
Probably want to omit `build_style` altogether in this case. `fetch` will download the file but not extract it.

Reply:
Fixed. Thank you.